### PR TITLE
Update Broken links in KFServing documentation

### PR DESCRIPTION
**Pull request by tcnt**
Update links to SDK client example.
Update link to Knative monitoring. Replace it by a link to kfserving github, where the details can be found.
See issue https://github.com/kubeflow/website/issues/2329

### CODE REVIEW

File: content/en/docs/components/serving/kfserving.md
Comment (animeshsingh) [inline content/en/docs/components/serving/kfserving.md:77]:
```suggestion
* [SDK client](https://github.com/kubeflow/kfserving/blob/master/docs/samples/client/kfserving_sdk_v1alpha2_sample.ipynb)
```

Comment (tcnt) [reply content/en/docs/components/serving/kfserving.md:77]: @animeshsingh I had it on purpose on the directory level, since there are now 2 notebooks.
By using the directory, the link does not need to change if the underlying notebook(s) change again.
I did the same in the readme on kfserving kubeflow/kfserving#1193 which was approved by @yuzisun

Comment (animeshsingh) [reply content/en/docs/components/serving/kfserving.md:77]: hear you - but the website docs point to an official release, and that section is title "notebooks" specifically. The 2nd notebook is still not official for overall Kubeflow

Comment (tcnt) [reply content/en/docs/components/serving/kfserving.md:77]: ok. Clear. But should I then not also change the same link in the SDK Section?

Comment (animeshsingh) [reply content/en/docs/components/serving/kfserving.md:77]: it's fine there. no strong opinion

File: content/en/docs/components/serving/kfserving.md
Comment (8bitmp3) [inline content/en/docs/components/serving/kfserving.md:26]: - For a11y + rephrase
```suggestion
Check the examples running KFServing on Istio/Dex in the [`kubeflow/kfserving`](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) repository. For installation on major cloud providers with Kubeflow, follow their installation docs.
```

Would it be possible to find links to AWS etc KFServing installation guides and include them here? This can help new users 💯, I think

File: content/en/docs/components/serving/kfserving.md
Comment (8bitmp3) [inline content/en/docs/components/serving/kfserving.md:115]:
```suggestion
1. Install the SDK with PiPy.

    ```shell
    pip install kfserving
    ```

```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
PTAL @tcnt 👍 

Thank you for the help!

File: content/en/docs/components/serving/kfserving.md
Comment (tcnt) [reply content/en/docs/components/serving/kfserving.md:26]: The text above refers to follow the installation docs of the major cloud providers.
Examples should then be in the existing links like https://www.kubeflow.org/docs/aws/
I do not think we should additional links on the current page.

**Comment by 8bitmp3** [comment]
/lgtm 

thanks @tcnt 

/assign @adrian555 @animeshsingh @cliveseldon @ellistarn @ryandawsonuk @Tomcli @yuzisun
(OWNERS-approvers)

**Comment by animeshsingh** [comment]
@tcnt looks like you need to rebase before we merge

**Comment by tcnt** [comment]
> @tcnt looks like you need to rebase before we merge

@animeshsingh I actually had to solve a merge conflict, as 1 of the lines I updated was also updated in the pull request https://github.com/kubeflow/website/pull/2342 which was merged into the master yesterday.
As far as I can see it should be ok now

**Comment by iancoffey** [comment]
Unfortunately, looks like this PRs updates still direct folks to broken 404ed monitoring resources - try the metrics link in the github markdown, for example: https://github.com/kubeflow/kfserving#setup-monitoring

I think this monitoring situation still need a solution, and hope we can discuss inhttps://github.com/kubeflow/kfserving/pull/1162 as this is still a dead-end afaict.

**Comment by tcnt** [comment]
@yuzisun Can you please have a look at this? The changes made in the current pull request where made assuming https://github.com/kubeflow/kfserving/pull/1162 would merge soon as well. As we discussed in https://github.com/kubeflow/website/issues/2329 you proposed to make the changes to refer to [KFS monitoring section](https://github.com/kubeflow/kfserving#setup-monitoring)

**Pull request merged**
